bucket/free-manga-downloader-2.json

```
@@ -0,0 +1,55 @@
+{
+    "version": "2.0.31.0",
+    "description": "an active fork of the Free Manga Downloader which is a free open source application written in Object Pascal for managing and downloading manga from various websites.",
```

```suggestion
    "description": "Active fork of Free Manga Downloader, an application for managing and downloading manga from various websites.",
```

Also rename the manifest to `free-manga-downloader2.json`